fuzzers/klee/fuzzer.py

```diff
@@ -66,13 +67,23 @@ def emptydir(path):
     os.mkdir(path)
 
 
-def run(command, hide_output=False):
-    """Run a the command |command|"""
+def run(command, hide_output=False, ulimit_cmd=None):
```

Comment:
Why is this the ulimit stuff needed?

Reply:
STP uses recursion when constraint solving. If you have a stack limit KLEE reaches it quite quickly and dies. However I'm not sure if these work inside docker, usually you have to tell docker, that the container has no stack limit.

Reply:
As @kren1 mentioned, this is necessary for STP. 

I've tested this with both libpng and re2 (fixes for those specific benchmarks coming in another PR) and this fixed the issue.

I had set the unlimit in the dockerfile, but it didn't appear to take. This was the only way I was able to clear up the issue with STP. Open to suggestions if anyone has a cleaner way to do this.